### ISSUE TITLE
Bulk edit is not supported on Tiles or Social views

### ISSUE THREAD

**Issue by merefield** [angusmcleod/discourse-topic-previews#46]
Bulk edit is not supported on Tiles or Social views

**Comment by angusmcleod** [comment]
@merefield Duplicate of https://github.com/angusmcleod/discourse-topic-previews/issues/47 ?

**Comment by merefield** [comment]
No, one was simply a case of the bulk edit menu control missing, this issue relates to the inability to bring up the checkboxes.

**Comment by merefield** [comment]
No longer relevant for sidecar
